pom.xml

```diff
@@ -87,37 +87,36 @@
     <!-- ================ -->
     <!-- Plugins versions -->
     <!-- ================ -->
-    <version.assembly.plugin>3.1.1</version.assembly.plugin>
-    <version.clean.plugin>3.2.0</version.clean.plugin>
-    <version.compiler.plugin>3.8.1</version.compiler.plugin>
-    <version.dependency.plugin>3.1.1</version.dependency.plugin>
-    <version.deploy.plugin>3.0.0-M1</version.deploy.plugin>
-    <version.enforcer.plugin>3.1.0</version.enforcer.plugin>
-    <version.surefire.plugin>2.22.2</version.surefire.plugin>
+    <version.assembly.plugin>3.6.0</version.assembly.plugin>
+    <version.clean.plugin>3.3.1</version.clean.plugin>
+    <version.compiler.plugin>3.11.0</version.compiler.plugin>
+    <version.dependency.plugin>3.6.0</version.dependency.plugin>
+    <version.deploy.plugin>3.1.1</version.deploy.plugin>
+    <version.enforcer.plugin>3.3.0</version.enforcer.plugin>
+    <version.surefire.plugin>3.1.2</version.surefire.plugin>
     <version.failsafe.plugin>${version.surefire.plugin}</version.failsafe.plugin>
-    <version.install.plugin>3.0.0-M1</version.install.plugin>
+    <version.install.plugin>3.1.1</version.install.plugin>
     <version.beanshell.plugin>1.4</version.beanshell.plugin>
-    <version.jar.plugin>3.1.2</version.jar.plugin>
-    <version.jarjar.plugin>1.9</version.jarjar.plugin>
-    <version.javadoc.plugin>3.1.0</version.javadoc.plugin>
-    <version.plugin.plugin>3.6.0</version.plugin.plugin>
-    <version.resources.plugin>3.1.0</version.resources.plugin>
-    <version.shade.plugin>3.2.1</version.shade.plugin>
-    <version.source.plugin>3.1.0</version.source.plugin>
+    <version.jar.plugin>3.3.0</version.jar.plugin>
+    <version.javadoc.plugin>3.5.0</version.javadoc.plugin>
+    <version.plugin.plugin>3.9.0</version.plugin.plugin>
+    <version.resources.plugin>3.3.1</version.resources.plugin>
+    <version.shade.plugin>3.5.0</version.shade.plugin>
+    <version.source.plugin>3.3.0</version.source.plugin>
     <version.site.plugin>3.7.1</version.site.plugin>
-    <version.artifactory.plugin>3.4.0</version.artifactory.plugin>
-    <version.jacoco.plugin>0.8.8</version.jacoco.plugin>
+    <version.artifactory.plugin>3.6.1</version.artifactory.plugin>
```

Comment:
This is a breaking change

Reply:
Should a separate ticket be created to investigate what should be fixed? Not being able to update a plugin is dangerous in the long run.

Reply:
For artifactory, yes this needs dedicated work as it's widely used and they apparently changed some behaviors with the environment variables.